### PR TITLE
Added NCS20071, NCS20072 and NCS20074

### PR DESCRIPTION
**Pull request by JeppeSRC**
[Datasheet](https://www.mouser.se/datasheet/2/308/NCS20071-D-470030.pdf)

img
img2
img3

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.

Violation S3.6: Pin name position offset is set to 0,127 because it's like that on every other OpAmp symbol.

### CODE REVIEW

**Comment by chschlue** [comment]
- [ ] Leave out packag**ing** and Pb-free suffixes, see https://www.onsemi.com/pub/Collateral/TND310-D.PDF#page=2
- [ ] Use manufacturer doc links, i.e. https://www.onsemi.com/pub/Collateral/NCS20071-D.PDF
- [ ] Convert dual and quad variants to [fully specified](http://kicad-pcb.org/libraries/klc/G2.1/)
- [ ] Put commas before packages in the descriptions
- [ ] Check for options to use aliases instead of additional full-fledged symbols

**Comment by chschlue** [comment]
@JeppeSRC ping

**Comment by JeppeSRC** [comment]
Oh damn, I forgot about this sorry!

**Comment by JeppeSRC** [comment]
@chschlue

File: Amplifier_Operational.lib [hunk 1]
```diff
@@ -1299,14 +1299,110 @@ X - 4 -300 -100 100 R 50 50 1 1 I
...
 #
+# NCS20071SN2T
+#
+DEF NCS20071SN2T U 0 5 Y Y 1 F N
+F0 "U" 0 400 50 H V C CNN
+F1 "NCS20071SN2T" 0 300 50 H V C CNN
+F2 "Package_SO:TSOP-5_1.65x3.05mm_P0.95mm" 50 1200 50 H I C CNN
+F3 "" 50 1200 50 H I C CNN
+$FPLIST
+ TSOP*1.65x3.05mm?P0.95mm*
```
Comment (chschlue) [inline Amplifier_Operational.lib]: TSOP\*1.65x3.05mm\*P0.95mm\*

File: Amplifier_Operational.lib [hunk 2]
```diff
@@ -1299,14 +1299,110 @@ X - 4 -300 -100 100 R 50 50 1 1 I
 ENDDRAW
 ENDDEF
 #
+# NCS20071SN2T
+#
+DEF NCS20071SN2T U 0 5 Y Y 1 F N
```
Comment (chschlue) [inline Amplifier_Operational.lib]: NCS20071SN

Package designator is SN, rest is packaging stuff

File: Amplifier_Operational.lib [hunk 3]
```diff
@@ -1299,14 +1299,110 @@ X - 4 -300 -100 100 R 50 50 1 1 I
...
+X V- 2 -100 -300 150 U 50 50 1 1 W
+X + 3 -300 100 100 R 50 50 1 1 I
+X - 4 -300 -100 100 R 50 50 1 1 I
+X V+ 5 -100 300 150 D 50 50 1 1 W
+ENDDRAW
+ENDDEF
+#
+# NCS20071XV53T
+#
+DEF NCS20071XV53T U 0 5 Y Y 1 F N
```
Comment (chschlue) [inline Amplifier_Operational.lib]: NCS20071XV

File: Amplifier_Operational.lib [hunk 4]
```diff
@@ -1299,14 +1299,110 @@ X - 4 -300 -100 100 R 50 50 1 1 I
...
+X V- 2 -100 -300 150 U 50 50 1 1 W
+X - 3 -300 -100 100 R 50 50 1 1 I
+X ~ 4 300 0 100 L 50 50 1 1 O
+X V+ 5 -100 300 150 D 50 50 1 1 W
+ENDDRAW
+ENDDEF
+#
+# NCS20072DTBR
+#
+DEF NCS20072DTBR U 0 5 Y Y 3 L N
```
Comment (chschlue) [inline Amplifier_Operational.lib]: NCS20072DTB

File: Amplifier_Operational.lib [hunk 5]
```diff
@@ -1299,14 +1299,110 @@ X - 4 -300 -100 100 R 50 50 1 1 I
...
+F0 "U" 100 250 50 H V L CNN
+F1 "NCS20072DTBR" 100 150 50 H V L CNN
+F2 "Package_SO:TSSOP-8_4.4x3mm_P0.65mm" 100 0 50 H I C CNN
+F3 "" 250 150 50 H I C CNN
+$FPLIST
+ TSSOP*8*4.4x3mm*P0.65mm*
```
Comment (chschlue) [inline Amplifier_Operational.lib]: TSSOP\*4.4x3mm\*P0.65mm\*

File: Amplifier_Operational.lib [hunk 6]
```diff
@@ -1299,14 +1299,110 @@ X - 4 -300 -100 100 R 50 50 1 1 I
...
+X - 6 -300 -100 100 R 50 50 2 1 I
+X ~ 7 300 0 100 L 50 50 2 1 O
+X V- 4 0 -300 150 U 50 50 3 1 W
+X V+ 8 0 300 150 D 50 50 3 1 W
+ENDDRAW
+ENDDEF
+#
+# NCS20074DTBR
+#
+DEF NCS20074DTBR U 0 5 Y Y 5 L N
```
Comment (chschlue) [inline Amplifier_Operational.lib]: NCS20074DTB

File: Amplifier_Operational.lib [hunk 7]
```diff
@@ -1299,14 +1299,110 @@ X - 4 -300 -100 100 R 50 50 1 1 I
...
+F0 "U" 100 250 50 H V L CNN
+F1 "NCS20074DTBR" 100 150 50 H V L CNN
+F2 "Package_SO:TSSOP-14_4.4x5mm_P0.65mm" 100 0 50 H I C CNN
+F3 "" 250 150 50 H I C CNN
+$FPLIST
+ TSSOP*14*4.4x5mm*P0.65mm*
```
Comment (chschlue) [inline Amplifier_Operational.lib]: TSSOP\*4.4x5mm\*P0.65mm\*

File: Amplifier_Operational.lib [hunk 8]
```diff
@@ -1299,14 +1299,110 @@ X - 4 -300 -100 100 R 50 50 1 1 I
...
+#
 # NCS2325D
 #
 DEF NCS2325D U 0 5 Y Y 3 L N
 F0 "U" 100 250 50 H V L CNN
 F1 "NCS2325D" 100 150 50 H V L CNN
 F2 "Package_SO:SOIC-8_3.9x4.9mm_P1.27mm" 100 0 50 H I C CNN
 F3 "" 250 150 50 H I C CNN
-ALIAS OPA2197xD OPA2196xD AD8676xR OPA1692xD
+ALIAS OPA2197xD OPA2196xD AD8676xR OPA1692xD NCS20072DR
```
Comment (chschlue) [inline Amplifier_Operational.lib]: NCS20072D

File: Amplifier_Operational.lib [hunk 9]
```diff
@@ -1375,6 +1472,7 @@ F0 "U" 100 250 50 H V L CNN
 F1 "NCS4325" 100 150 50 H V L CNN
 F2 "Package_SO:SOIC-14_3.9x8.7mm_P1.27mm" 100 0 50 H I C CNN
 F3 "" 250 150 50 H I C CNN
+ALIAS NCS20074DR
```
Comment (chschlue) [inline Amplifier_Operational.lib]: NCS20074D

File: Amplifier_Operational.lib [hunk 10]
```diff
@@ -1331,6 +1427,7 @@ F0 "U" 100 250 50 H V L CNN
 F1 "NCS2325DM" 100 150 50 H V L CNN
 F2 "Package_SO:MSOP-8_3x3mm_P0.65mm" 100 0 50 H I C CNN
 F3 "" 250 150 50 H I C CNN
+ALIAS NCS20072DMR
```
Comment (chschlue) [inline Amplifier_Operational.lib]: NCS20072DM

**Comment by chschlue** [comment]
Thanks.

**Pull request merged**